kafka/config.go

```diff
@@ -183,6 +183,14 @@ func (m ConfigMap) convert() (cConf *C.rd_kafka_conf_t, err error) {
 // If the key is not found defval is returned.
 // If the key is found but the type is mismatched an error is returned.
 func (m ConfigMap) get(key string, defval ConfigValue) (ConfigValue, error) {
+	if strings.HasPrefix(key, "{topic}.") {
```

Comment:
is this the first client where we de-nest topic configuration?

have we considered (and is it worth having a plan for) how this might generalize to specific topic config? what you have could work I think, but i feel like I want to see this worked through before committing to anything. I'm not sure about the notation `{topic}` but not immediately having better ideas.